src/Umbraco.Core/Services/ContentTypeService.cs

```diff
@@ -82,6 +83,26 @@ public IEnumerable<int> GetAllContentTypeIds(string[] aliases)
         }
     }
 
+    /// <inheritdoc />
+    public Task<PagedModel<IContentType>> GetAllAllowedAsRootAsync(int skip, int take)
+    {
+        using ICoreScope scope = ScopeProvider.CreateCoreScope(autoComplete: true);
+
+        // that one is special because it works across content, media and member types
```

Comment:
I don't understand this. IMO should we have explicit methods for content, media and members that only locks that tree.